assets/dev/js/editor/regions/navigator/elements/hooks/ui/base/base-hide-show.js

```diff
@@ -0,0 +1,9 @@
+import Base from './base';
+
+export default class BaseHideShow extends Base {
+	apply( args ) {
+		const { containers = [ args.container ] } = args;
+
+		containers.forEach( ( container ) => container.view.toggleVisibilityClass() );
```

Comment:
You should hide or show. not toggle the state. (`hide` for a hidden element should not show it)
(Fix also the show and hide command)

Reply:
Handled.